src/scss/custom/components/misc/_header-block.scss

```diff
@@ -0,0 +1,70 @@
+$component-name: header-block;
+
+.#{$component-name} {
+  position: relative;
+
+  &__action-btn {
+    display: inline-flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    padding: 0.75rem 0.25rem;
+    color: var(--header-top-color);
+    border-radius: $btn-border-radius;
+
+    &:hover,
+    &:focus {
+      color: var(--header-top-color);
+    }
+
+    @include media-breakpoint-up(md) {
+      padding: $btn-padding-y $btn-padding-x;
```

Comment:
I guess we don't have any `CSS variables` to consume there?

Reply:
Thanks @NeOMakinG for your review, changes applied.